frontend/packages/console-shared/src/utils/multiselectdropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = (props) => {
+  const { onChange, placeholder, id } = props;
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState([]);
+  const onToggle = (open: boolean) => {
+    setIsOpen(open);
+  };
+  const onSelect = (event: React.MouseEvent | React.ChangeEvent, selection: string) => {
+    let cSelected: string[] = selected;
+    if (selected.includes(selection)) {
+      cSelected = selected.filter((item) => item !== selection);
+    } else {
+      cSelected = [...selected, selection];
+    }
+    setSelected(cSelected);
+    onChange(cSelected);
+  };
+  const items: JSX.Element[] = props.options.map((item) => {
+    return <SelectOption key={0} value={item} />;
+  });
+  return (
+    <div>
+      <span id={id} hidden>
+        Checkbox Title
+      </span>
+
+      <Select
+        variant={SelectVariant.typeaheadMulti}
+        aria-label="Select Input"
+        onToggle={onToggle}
```

Comment:
If we are just setting state, we don't need to define another function.
```suggestion
        onToggle={setIsOpen}
```

Reply:
done